armoryengine/Transaction.py

```diff
@@ -648,6 +651,55 @@ def toString(self, nIndent=0, endian=BIGENDIAN):
       return result
 
 #####
+class PyTxWitness(BlockComponent):
+   def __init__(self):
+      self.binWitness  = UNINITIALIZED
+
+   def unserialize(self, toUnpack):
+      if isinstance(toUnpack, BinaryUnpacker):
+         txWitnessData = toUnpack
+      else:
+         txWitnessData = BinaryUnpacker( toUnpack )
+
+      self.binWitness = []
+      stackSize = txWitnessData.get(VAR_INT)
+      self.binWitness.append(stackSize)
+      if stackSize > 0:
+         for i in range(0, stackSize, 1):
+            stackItemSize = txWitnessData.get(VAR_INT)
+            debugremsize = txWitnessData.getRemainingSize()
+            if txWitnessData.getRemainingSize() < stackItemSize: raise UnserializeError
+            stackItem = txWitnessData.get(BINARY_CHUNK, stackItemSize)
+            self.binWitness.append(stackItemSize)
```

Comment:
I dont believe this will put stackItemSize as a varint in self.binWitness

You are better off copying straight from txWitnessData